nanithing-jaxrs-interfaces/pom.xml

```diff
@@ -1,45 +1,48 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-	<modelVersion>4.0.0</modelVersion>
-	
-	<parent>
-		<groupId>nanithing</groupId>
-		<artifactId>nanithing-parent</artifactId>
-		<version>1.0.0-SNAPSHOT</version>
-	</parent>
-	<artifactId>nanithing-jaxrs-interfaces</artifactId>
-	
-	<name>Nanithing: JAX-RS interfaces</name>
-	<description>Nanithing: JAX-RS interfaces</description>
-	
-	<dependencies>
-		<!-- SPECS -->
-		<dependency>
-			<groupId>org.jboss.spec.javax.ws.rs</groupId>
-			<artifactId>jboss-jaxrs-api_2.1_spec</artifactId>
-		</dependency>
-		
-		<!-- PROJECT -->
-		<dependency>
-			<groupId>nanithing</groupId>
-			<artifactId>nanithing-model</artifactId>
-			<version>${project.version}</version>
-		</dependency>
-	</dependencies>
-	
-	<build>
-		<plugins>
-			<plugin>
-				<groupId>org.jboss.jandex</groupId>
-				<artifactId>jandex-maven-plugin</artifactId>
-				<executions>
-					<execution>
-						<id>make-index</id>
-						<goals>
-							<goal>jandex</goal>
-						</goals>
-					</execution>
-				</executions>
-			</plugin>
-		</plugins>
-	</build>
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>nanithing</groupId>
+    <artifactId>nanithing-parent</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>nanithing-jaxrs-interfaces</artifactId>
+  <name>Nanithing: JAX-RS interfaces</name>
+  <description>Nanithing: JAX-RS interfaces</description>
+  <dependencies>
+    <!-- SPECS -->
+    <dependency>
+      <groupId>org.jboss.spec.javax.ws.rs</groupId>
+      <artifactId>jboss-jaxrs-api_2.1_spec</artifactId>
+    </dependency>
+    <!-- PROJECT -->
+    <dependency>
+      <groupId>nanithing</groupId>
+      <artifactId>nanithing-model</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <!-- OPENAPI -->
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-smallrye-openapi</artifactId>
```

Comment:
No need to "tie" this artifact with `quarkus-smallrye-openapi`; depending on `org.eclipse.microprofile.openapi:microprofile-openapi-api` is sufficient and more reusable.